datafusion/sqllogictest/test_files/coalesce.slt

```diff
@@ -0,0 +1,374 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Given the size of these tests I opted to put them in their own file rather than `function.slt`